MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include kinto/plugins/admin/build *
```

Comment:
This is now managed here https://github.com/Kinto/kinto/blob/b67c6c32c71bfdc6f68c84e8da10b2fad4a54544/pyproject.toml#L54-L55

Reply:
Ah, that explains why the index.html file was showing up but nothing else.